build/.vsts-PR-internal-check-healthcare-shared-components-azureBuild-pipeline.yml

```diff
@@ -0,0 +1,149 @@
+name:  pr$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:-r)
```

Comment:
I think we can reduce the number of the checks we run on our code as they probably haven't been re-evaluated in awhile. For example, in DICOM we use the following: https://github.com/microsoft/dicom-server/blob/main/build/analyze.yml

And since we're running on Windows, you don't need to do the copy work we do to take the binaries we built on Linux machines.

Reply:
@wsugarman Great suggestion. I tried to look into reducing the number of checks.... just excluded security analysis check BinSkim as per you suggestion. Could not change the  pool name as checks are configured to run on windows machine.